scripts/Global/Global.gd

```diff
@@ -1,6 +1,8 @@
 extends Node
 
 var node_creation_parent = null
+var player = null 
```

Comment:
maybe add a class for the player and check if the player variable here is actually a player